SPEC.md

```diff
@@ -116,34 +119,62 @@ In this case, the app can reference the name "example.com/trusted-certificate-au
 The dependencies are applied in order and each image dependency can overwrite files from the previous dependency.
 An optional *path whitelist* can be provided, in which case all non-specified files from all dependencies will be omitted in the final, assembled rootfs.
 
-Image Format TODO
+## App Container Executor
 
-* Define the garbage collection lifecycle of the container filesystem including:
+The **App Container Executor** defines the process by which applications contained in ACIs are executed.
+There are two "perspectives" in this process.
+The "*executor*" perspective consists of the steps that the App Container Executor (ACE) must take to set up the environment for the pod and applications.
+The "*app*" perspective is how the app processes inside the pod see the environment.
+
+## Pods
+
+The deployable, executable unit in the App Container specification is the **pod**.
+A **pod** is a list of apps that should be launched together inside a shared context.
+The shared context is defined as the conjunction of the following Linux namespaces (or equivalents on other operating systems):
+- PID namespace
+- network namespace
+- mount namespace
+- IPC namespace
+- UTS namespace
+The context may also consist of one or more isolators.
+
+Each app in a pod will start chrooted into its own unique read-write rootfs before execution.
+
+The definition of the **pod** - namely, the list of constituent apps, and any isolators that should apply to the entire pod - is codified in a [Pod Manifest](#pod-manifest-schema).
```

Comment:
Fair enough, naming is hard and I do not have a strong track record
(lmctfy) :)
On Feb 21, 2015 11:14 AM, "Jonathan Boulle" <notifications@github.com>
wrote:

> In SPEC.md <https://github.com/appc/spec/pull/207#discussion_r25124735>:
>
> > +
> > +## Pods
> > +
> > +The deployable, executable unit in the App Container specification is the **pod**.
> > +A **pod** is a list of apps that should be launched together inside a shared context.
> > +The shared context is defined as the conjunction of the following Linux namespaces (or equivalents on other operating systems):
> > +- PID namespace
> > +- network namespace
> > +- mount namespace
> > +- IPC namespace
> > +- UTS namespace
> > +The context may also consist of one or more isolators.
> > +
> > +Each app in a pod will start chrooted into its own unique read-write rootfs before execution.
> > +
> > +The definition of the **pod** - namely, the list of constituent apps, and any isolators that should apply to the entire pod - is codified in a [Pod Manifest](#pod-manifest-schema).
>
> Hrm, we use the term "Image Manifest" for the JSON blob which describes a
> single image, not a list of images, so this is internally consistent. I am
> struggling to think of another term right now.
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/appc/spec/pull/207/files#r25124735>.
>